### ISSUE TITLE
modx user group & role

### ISSUE THREAD

**Issue by hiwawa** [sepiariver/auth0#2]
Hello,
new user is default NULL userGroup & role.

**Comment by sepiariver** [comment]
Correct. The Extra will not assume what permissions you want to give newly created Users. Should there be a System Setting to define a default User Group?

**Comment by sepiariver** [comment]
Note: it is possible to give users access to front-end Resources without them being added to any User Group.
